src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java

```diff
@@ -42,7 +43,7 @@
     public boolean escapeHtml = false;
 
     @Parameter
-    public String testSuiteName = "";
+    public String testNameRegexPattern = "";

```

Comment:
Not to nitpick, but I think just `testNamePattern` would be fine. This is kind of the naming convention used in various parts of Jenkins for where a regex can be used.